JS/Lesson30/tugba/my-app/src/components/Ex8/index.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import './index.css'
+export const Ex8 = () => {
+  const initialList = [
+    { id: 1, label: "Working hard" },
+    { id: 2, label: "Go to work" },
+    { id: 3, label: "Do dishes" },
+    { id: 4, label: "Attend meetings" },
+    { id: 5, label: "Being lazy" },
+    { id: 6, label: "Being happy" },
+    { id: 7, label: "Cook meal" },
+  ];
+
+  const [list, setList] = useState(initialList);
+
+  const deleteButton = (itemId) => {
+    const updatedList = list.filter((item) => item.id !== itemId);
+    console.log(itemId);
+    console.log(updatedList);
+    return setList(updatedList);
+  };
+
+  return (
+    <>
```

Comment:
You only need to use Fragment component  (`<></>`) in React if you have more than one element on the same level that you need to return. Here you only have `<ul>` so no need to wrap it in `<></>`.